File: C_module/Assignment 9/Q6.c
#include <stdio.h>

void prime(int*,int);
void main() {
    int n;
    printf("Size of array :: ");
    scanf("%d",&n);
    int arr[n];
    printf("Enter array Elements :: ");
    for(int i=0;i<n;i++){
        scanf("%d",&arr[i]);
    }
    prime(arr,n);
}

void prime(int* arr,int n){
    printf("Prime Elements are :: ");
    for (int i = 0; i < n; i++) {
        if (arr[i] < 2) continue;
        int flag = 0;

        for (int j = 2; j * j <= arr[i]; j++) {
            if (arr[i] % j == 0) {
                flag = 1;
                break;
            }
        }

        if (flag == 0) {
            printf("%d ", arr[i]);
        }
    }

}

File: Core_Java_module/Practice/9-16-2025 (Array Practice)/Q6.java
// Accept array and print only prime numbers of array.

import java.util.Scanner;

class Q6 {
    public static void main(String[] args) {
        int n;
		System.out.print("Enter array size : ");
		Scanner sc=new Scanner(System.in);
		n=sc.nextInt();
		int [] arr=new int [n];
		for(int i=0;i<n;i++){
			System.out.print("Enter "+(i+1)+" Element of Array : ");
			arr[i]=sc.nextInt();
		}		
		System.out.print("Array Elements are :");
		for(int i=0;i<n;i++){
			System.out.print(arr[i]+" , ");
		}
		System.out.println();
		System.out.print("Prime Numbers are");

        for(int i=0;i<n;i++){
            boolean isPrime=true;
            for(int j=2;j<arr[i]/2;j++){
                if(arr[i]%j == 0){
                    isPrime=false;
                    break;
                }
            }
            if (isPrime==true) {
                System.out.print(arr[i]+" , ");
            }
        }
    }
}